src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -35,7 +36,9 @@ public AutocheckingRecheckDriver( final RemoteWebDriver wrapped ) {
 	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped, final RecheckOptions options ) {
 		super( wrapped );
 		this.options = options;
-		checkNamingStrategy = RecheckWebOptions.builder().build().getCheckNamingStrategy();
+		final RecheckWebOptions defaultOptions = RecheckWebOptions.builder().build();
```

Comment:
Just realizing, that we should propably do an `instanceof RecheckWebOptions` here, if someone used
```
RecheckOptions opts = RecheckWebOptions.builder()
    ...
    .setAutocheckingDelayMillis( 500 )
    .build()
WebDriver driver = new AutocheckingRecheckDriver(chrome, opts);
```
Like:
```suggestion
		final RecheckWebOptions defaultOptions = options instanceof RecheckWebOptions ? (RecheckWebOptions) options : RecheckWebOptions.builder().build();
```

Reply:
You are right, I wasn't sure, but this failed:
```java
	private String t( final RecheckWebOptions o ) {
		return "web";
	}

	private String t( final RecheckOptions o ) {
		return "plain";
	}

	@Test
	void testName() throws Exception {
		final RecheckOptions o = RecheckWebOptions.builder().build();
		assertEquals( "web", t( o ) );
	}
```
I will make another commit to fix this.